src/chocolatey/infrastructure/platforms/Platform.cs

```diff
@@ -85,7 +85,7 @@ public static string get_name()
                 case PlatformType.Linux:
                     return "Linux";
                 case PlatformType.Mac:
-                    return "OS X";
+                    return "macOS";

```

Comment:
Initially we were concerned that this change would be a breaking change, however, looking at the various code paths in place, the decision was taken that this won't impact anyone, and the change makes sense in line with the branding of this OS.